### PR TITLE
Created hk-form web component

### PR DESCRIPTION
**Pull request by cferdinandi**
@dwinston This is "done," but would be improved by one routing update, if you'll humor me.

In the `main.py` file, can we change the redirect for `create_concept` (line 59) to point to the `read_concepts` route instead of the individual `read_concept` route?

I tried making the update myself, but could not.

### CODE REVIEW

**Comment by cferdinandi** [comment]
I should also have mentioned, this is for https://github.com/polyneme/heliokos/issues/16, and results in a substantially smaller and more resilient JavaScript payload plus an easier authoring experience.

It's similar to what you were doing with HTMX, but resolves the accessibility issues.

File: src/heliokos/ui/main.py [hunk 1]
```diff
@@ -61,7 +61,7 @@ async def create_concept(pref_label: Annotated[str, Form()], request: Request):
     concept.to_file()
     return RedirectResponse(
         status_code=status.HTTP_303_SEE_OTHER,
-        url=request.url_for("read_concept", id_=concept.id_suffix),
+        url=request.url_for("read_concepts"),
```
Comment (dwinston) [inline src/heliokos/ui/main.py:64]: made the change you wanted (I think).

**Review by dwinston — APPROVED** [review]
great! I like reading your code. very "future me"-friendly.

**Comment by cferdinandi** [comment]
> great! I like reading your code. very "future me"-friendly.

That was the idea! Glad you approve!

**Pull request merged**